### ISSUE TITLE
MediaNode attributes parameter conflicts with volume setter

### ISSUE THREAD

**Issue by Sacharified** [bbc/VideoContext#97]
When constructing a video/audio node, you can pass the argument `videoElementAttributes`. This is currently the only way to initialise the node with a volume value other than 1.0. 

`const node = context.video("video.mp4", 0, 0, { volume: 0.5 })`

If you then wish to modify the volume later on, you can use the `volume` setter, which modifies the `_volume` property of the node, but not `_attributes`.

During playback, the node's `_load` method will apply all of the properties in `_attributes` to the element, overriding the set volume with the value the node was constructed with.

To reproduce:

```
var canvas = document.getElementById("canvas");
var vc = new VideoContext(canvas);

var videoNode1 = vc.video("../../assets/introductions-rant.mp4", 0, 0, { volume: 1 });
videoNode1.connect(vc.destination);
videoNode1.start(0);
videoNode1.stop(10);

vc.play();
videoNode1.volume = 0.5;
```

The expected behaviour is that the element volume is set to 0.5 and stays there.

What actually happens is that the volume is set correctly, then the next time `_load` is called in the RAF callback, the element's volume is overwritten by the volume from the `_attributes` property.

**Comment by PTaylour** [comment]
Hi @Sacharified thanks you for reporting this. I agree that the volume setter should persist between calls to `_load`.


Something like this to fix: https://github.com/bbc/VideoContext/pull/98

**Comment by PTaylour** [comment]
Fix published in 0.52.7
